README.md

```diff
@@ -278,7 +211,7 @@ I would like to thank...
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](https://github.com/abdallahmalima/Bookstore/blob/dev/MIT.md) licensed.
```

Comment:
- [ ] @abdallahmalima great work in implementing the changes requested, kindly customize the [MIT](https://github.com/abdallahmalima/Bookstore/blob/dev/MIT.md) license with your info and the app info.